atm.js

```diff
@@ -1,21 +1,54 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+//Checking account
+//Define players
+$(document).ready(function() {
+
+var atm = {
+  accounts: {
+    checking: 0,
+    savings: 0
+  },
+  listen: function(){
+    $(".account").on("click", function() {
+      event.preventDefault();
+      var actionType = $(event.target).val();//Deposit of withdraw
+      var accountType = $(event.target).prevAll("h2").html();//Checking or savings
+      var amount = parseInt($(event.target).prevAll("[type=text]").val());//amount
+      if( actionType === "Deposit"){
+        this.deposit(accountType,amount)
+      }
+      if( actionType === "Withdraw"){
+        this.withdraw(accountType,amount)
+      }
```

Comment:
or `actionType === "Deposit" ? this.deposit(accountType,amount) : this.withdraw(accountType,amount)`

turnary operators are cool but harder to read. just wanted to share!